src/Database/Factories/Factory.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Winter\Storm\Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory as BaseFactory;
+use Illuminate\Support\Str;
+
+/**
+ * @template TModel of \Illuminate\Database\Eloquent\Model
```

Comment:
What is this used for? Should it be pointing at the winter base model class?

Reply:
Not sure, I copied this from Laravel.

I tried removing it completely and things still work as expected. It's probably only used by the docblock generator.